CODE-OF-CONDUCT.md

```diff
@@ -1,5 +1,3 @@
 # Code of conduct
 
-Everyone is invited to participate in Mapbox’s open source projects and public discussions: we want to create a welcoming and friendly environment. Harassment of participants or other unethical and unprofessional behavior will not be tolerated in our spaces. The [Contributor Covenant](http://contributor-covenant.org) applies to all projects under the Mapbox organization and we ask that you please read [the full text](http://contributor-covenant.org/version/1/2/0/).
-
-You can learn more about our open source philosophy on [mapbox.com](https://www.mapbox.com/about/open/).
+TBD
```

Comment:
Not sure if the Maplibre team have thoughts on this, but didn't want to direct users to the Mapbox CoC unintentionally.